Polygon/doc/Polygon/Doxyfile.in

```diff
@@ -3,7 +3,7 @@
 PROJECT_NAME = "CGAL ${CGAL_DOC_VERSION} - 2D Polygons"
 EXAMPLE_PATH = ${CGAL_PACKAGE_DIR}/examples \
                ${CGAL_Stream_support_EXAMPLE_DIR}
-
+INPUT += ${CGAL_PACKAGE_INCLUDE_DIR}/CGAL
```

Comment:
I broke the CGAL Doxygen testsuite with that patch. `${CGAL_PACKAGE_INCLUDE_DIR}/CGAL` evaluates to `...something.../include//CGAL`, and Doxygen-1.8.4 does not like the double `/`.